.travis.yml

```
@@ -1,30 +1,52 @@
+version: ~> 1.0
 language: scala
 scala:
   - 2.12.11
 
-env:
-  matrix:
-    - TRAVIS_JDK=adopt@1.8.0-242
-    - TRAVIS_JDK=adopt@1.11.0-6
-
 before_install: curl -Ls https://git.io/jabba | bash && . ~/.jabba/jabba.sh
 install: jabba install "$TRAVIS_JDK" && jabba use "$_" && java -Xmx32m -version
 
-script:
- - sbt docs/paradox
- # `sbt test` already materializes the template and runs `test` on the materialized code.
- - sbt test
- ## set name to "hello-world-grpc" and accept all other defailts (--force)
- - sbt new file://$PWD --name=hello-world-grpc --force && pushd hello-world-grpc &&  mvn test         && popd || travis_terminate 1
- - sbt new file://$PWD --name=hello-world-grpc --force && pushd hello-world-grpc &&  ./gradlew build  && popd || travis_terminate 1
- 
+jobs:
+  include:
+    - stage: check
+      name: "sbt & Paradox (JDK 8)"
+      script: sbt test docs/paradox
+      env: TRAVIS_JDK=adopt@1.8.0-242
+
+    - name: "sbt & Paradox (JDK 11)"
+      script: sbt test docs/paradox
+      env: TRAVIS_JDK=adopt@1.11.0-6
```

```suggestion
      env: TRAVIS_JDK=adopt@1.11.0-7
```
(same in jobs below)